### PR TITLE
Added new tutorial: Setup ZFS Storage Pool

### PR DESCRIPTION
**Pull request by aden-chrome**

### CODE REVIEW

File: tutorials/server/setup-zfs-storage-pool/setup-zfs-storage-pool.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- Ubuntu Server 16.04 LTS
+
+Ready? Let's head over to the next step!
+
+## Installing ZFS
+Duration: 1:00
+
+To install ZFS, run:
+```bash
+sudo apt install zfs
```
Comment (evilnick) [inline tutorials/server/setup-zfs-storage-pool/setup-zfs-storage-pool.md]: Have you checked this? I thought the package name was zfsutils-linux

Comment (aden-chrome) [reply tutorials/server/setup-zfs-storage-pool/setup-zfs-storage-pool.md]: it worked for me during the run through but you're right, zfs isn't the package name, it does this redirect thing when you use it.

image

should i change it?

**Comment by aden-chrome** [comment]
@evilnick I've corrected the package name and updated the help link.

**Comment by evilnick** [comment]
@caldav want to give this a quick once over?

**Comment by evilnick** [comment]
thanks for the tweaks :bee:

**Pull request merged**